sql/moz-fx-data-shared-prod/stub_attribution_service/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Stub Attribution Service
+description: |-
+  Stub attribution service data, usually from the logs.
+dataset_base_acl: view_restricted
```

Comment:
@whd I moved to a new dataset so we don't have to restrict the GA data. Does this setup for the derived and view datasets look good on your end to restrict dev access to just Airflow (unless I later add ppl to tf) and query access to just the duet wg?

Reply:
This PR looks correct from an access perspective in that it matches the [uniform access archetype]( https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27924789/Data+Access+Workgroups#DataAccessWorkgroups-Exampledatasetwithuniformspecificworkgroupaccess(creatingadatasetthatisaccessibletoaspecificworkgroup)).

It isn't necessary to configure access this way, but it's probably the easiest to reason about. The other option is to leverage a single dataset with a restricted base ACL and specific workgroup-confidential tables configured and have `default_table_workgroup_access` set to `workgroup:mozilla-confidential` to allow for mixed ACL management. Table automation is all hooked up to support this and I'd be interested in seeing if it works (we haven't set up any deprecation configuration yet AFAIK) but splitting out the workgroup-confidential data like this PR does also works.